rel/env.bat.eex

```diff
@@ -0,0 +1,6 @@
+@echo off
+rem Set the release to work across nodes. If using the long name format like
+rem the one below (my_app@127.0.0.1), you need to also uncomment the
+rem RELEASE_DISTRIBUTION variable below. Must be "sname", "name" or "none".
+rem set RELEASE_DISTRIBUTION=name
+rem set RELEASE_NODE=<%= @release.name %>@127.0.0.1
```

Comment:
Hmm, how did these files end up here 🤔 
Please remove this from git. no need to push `rel` folder.